src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -234,47 +234,54 @@ public function __construct(FactoryInterface $factory, Connection $conn)
      */
     private function registerSqliteFunctions(PDO $sqliteConnection)
     {
-        $sqliteConnection->sqliteCreateFunction('EXTRACTVALUE', function ($string, $expression) {
-            if (null === $string) {
-                return null;
-            }
+        $sqliteConnection->sqliteCreateFunction(
+            'EXTRACTVALUE',
+            function ($string, $expression) {
+                if (null === $string) {
+                    return null;
+                }
 
-            $dom = new DOMDocument('1.0', 'UTF-8');
-            $dom->loadXML($string);
-            $xpath = new DOMXPath($dom);
-            $list = $xpath->evaluate($expression);
+                $dom = new DOMDocument('1.0', 'UTF-8');
+                $dom->loadXML($string);
+                $xpath = new DOMXPath($dom);
+                $list = $xpath->evaluate($expression);
 
-            if (!is_object($list)) {
-                return $list;
-            }
+                if (!is_object($list)) {
+                    return $list;
+                }
 
-            // @TODO: don't know if there are expressions returning more then one row
-            if ($list->length > 0) {
-                // @TODO: why it can happen that we do not have a type? https://github.com/phpcr/phpcr-api-tests/pull/132
-                $type = is_object($list->item(0)->parentNode->attributes->getNamedItem('type'))
-                    ? $list->item(0)->parentNode->attributes->getNamedItem('type')->value
-                    : null;
-                $content = $list->item(0)->textContent;
-
-                switch ($type) {
-                    case 'long':
-                        return (int) $content;
-                        break;
-                    case 'double':
-                        return (double) $content;
-                        break;
-                    default:
-                        return $content;
+                // @TODO: don't know if there are expressions returning more then one row
+                if ($list->length > 0) {
+                    // @TODO: why it can happen that we do not have a type? https://github.com/phpcr/phpcr-api-tests/pull/132
+                    $type = is_object($list->item(0)->parentNode->attributes->getNamedItem('type')) ? $list->item(
+                        0
+                    )->parentNode->attributes->getNamedItem('type')->value : null;
+                    $content = $list->item(0)->textContent;
+
+                    switch ($type) {
+                        case 'long':
+                            return (int)$content;
+                            break;
+                        case 'double':
+                            return (double)$content;
+                            break;
+                        default:
+                            return $content;
+                    }
```

Comment:
while i agree with the formatting, i think such changes would better be in a separate PR that is only about styling and not mixed with feature changes, as it makes it hard to see what the actual changes are ;-)

but no need to revert it this time, as its already committed.

Reply:
👍 no problem - i have a built in formatter in my ide so its hard to seperate it! but will take a look at it next time

Reply:
yeah i thought as much. for reviewing its not ideal, and neither for git history to figure out what was going on ;-)